packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -468,12 +1375,29 @@ export class FabricTestLedgerV1 implements ITestLedger {
     return Containers.stop(this.container as Container);
   }
 
-  public async destroy(): Promise<any> {
+  public async destroy(): Promise<void> {
     const fnTag = "FabricTestLedgerV1#destroy()";
-    if (this.container) {
-      return this.container.remove();
-    } else {
-      throw new Error(`${fnTag} Containernot found, nothing to destroy.`);
+    try {
+      if (this.container) {
+        const docker = new Dockerode();
+        const containerInfo = await this.container.inspect();
+        const volumes = containerInfo.Mounts;
+        await this.container.remove();
+        volumes.forEach(async (volume) => {
+          this.log.debug("Removing volume: ", volume);
+          if (volume.Name) {
+            const volumeToDelete = docker.getVolume(volume.Name);
+            await volumeToDelete.remove();
+          } else {
+            this.log.debug("Volume", volume, "could not be removed");
+          }
+        });
+      } else {
+        throw new Error(`${fnTag} Container not found, nothing to destroy.`);
+      }
```

Comment:
@RafaelAPB The indentations can be reduced here with equivalent logic being in place (which is good because it reduces the chance of arrow programming [1] occurring. Recommended change below is to throw if `this.container` is falsy and then there's no need to have the rest of the logic on an explicit `else` branch.

[1]: https://blog.codinghorror.com/flattening-arrow-code/

```suggestion
  if (!this.container) {
    throw new Error(`${fnTag} Container not found, nothing to destroy.`);
  }
  const docker = new Dockerode();
  const containerInfo = await this.container.inspect();
  const volumes = containerInfo.Mounts;
  await this.container.remove();
  volumes.forEach(async (volume) => {
    this.log.debug("Removing volume: ", volume);
    if (volume.Name) {
      const volumeToDelete = docker.getVolume(volume.Name);
      await volumeToDelete.remove();
    } else {
      this.log.debug("Volume", volume, "could not be removed");
    }
  });
```